src/sentry/static/sentry/less/components/charts.less

```diff
@@ -15,13 +15,16 @@
     cursor: default;
 
     &:hover {
-      > span {
+      > span,
```

Comment:
I'm gonna support both svg and html based charts until I can convert everything over to styled components in a follow-up pull.